specification/cognitiveservices/data-plane/QnAMaker/stable/v4.0/QnAMaker.json

```diff
@@ -671,6 +679,32 @@
         }
       }
     },
+    "UpdateContextDTO": {
+      "type": "object",
+      "description": "Update Body schema to represent context to be updated",
+      "additionalProperties": false,
+      "properties": {
+        "promptsToDelete": {
```

Comment:
Please mark the readonly properties where applicable

Reply:
These are all read/write